ui/pages/Project/components/FamilyTable/IndividualRow.jsx

```diff
@@ -169,7 +169,7 @@ const DataDetails = React.memo(({ loadedSamples, individual, mmeSubmission }) =>
         />
       ) : <MmeStatusLabel title="Submitted to MME" dateField="lastModifiedDate" color="violet" individual={individual} mmeSubmission={mmeSubmission} />
     )}
-    {individual.hasRnaOutlierData && (
+    {loadedSamples.some(sample => sample.rnaSeqTypes?.length > 0) && (
```

Comment:
`hasRnaOutlierData` was based on whether there was `rnaseqoutlier` or `rnaseqspliceoutlier` data, for samples with only tpm data this would have been false, so we should update the behavior here to reflect that. Also it only considered active samples

Reply:
I adjusted this condition check to match the behavior to show the link only if the sample is active and has either rnaseqoutlier or rnaseqspliceoutlier data